tensorrt_llm/_torch/models/modeling_utils.py

```diff
@@ -831,7 +831,7 @@ def load_single_module(name, module):
                             p.data.copy_(module_weights[n][:])
 
     if os.environ.get("TRT_LLM_DISABLE_LOAD_WEIGHTS_IN_PARALLEL",
-                      False) in ["True", "true", "1", "yes", "y"]:
+                      False) in ["False", "false", "0", "no", "n"]:
```

Comment:
I think we need to use `os.environ.get("TRT_LLM_DISABLE_LOAD_WEIGHTS_IN_PARALLEL", True)`?

current logic is has issues:
if TRT_LLM_DISABLE_LOAD_WEIGHTS_IN_PARALLEL==True, set by user, it will go to else branch? And still do concurrent loading?